capstone/fitness/src/main/webapp/calendar.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<!-- HTML for calendar page. Containers load in information from java using the onload in body. -->
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Your Calendar</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <!-- basic navbar for navigation during development. can be stylized later. -->
+  <div class="topnav">
+    <a class="active" href="/">Home</a>
+    <a href="/calendar.html">Calendar</a>
+    <a href="/">Progress</a>
+    <a href="/">Data</a>
+  </div>
+  <body onload="getCalendarInfo();">
```

Comment:
Can you add a simple implementation of getCalendarInfo() that gets data from CalendarServlet?

Reply:
Yes